docs/guides/B2B-Suite/b2b-organizations/bulk-import-spreadsheet.md

```diff
@@ -0,0 +1,97 @@
+# Bulk Import Spreadsheet
+
+This guide will walk you through the process of filling out the bulk import spreadsheet for use with the [Bulk Import API](https://developers.vtex.com/docs/api-reference/buyer-organizations) to efficiently manage Buyer Organizations, Cost Centers, and Members through bulk import functionality.
+
```

Comment:
Faltou linkar melhor com as demais documentações de B2B e adicionar uma seção `## Before you begin` para explicar que para seguir as intruções deste guia precisa ter os apps da B2B Suite instalados

Reply:
Adicionado